src/types/enums.ts

```diff
@@ -17,5 +17,13 @@ export enum RequestParameter {
   DATA_GRAPH_FILE = 'data-graph-file',
   DATA_GRAPH_URL = 'data-graph-url',
   SHAPES_GRAPH_FILE = 'shapes-graph-file',
+  SHAPES_GRAPH_URL = 'shapes-graph-url',
   CONFIG = 'config'
 }
+
+export enum InputType {
+  FILE = 'file',
+  URL = 'url',
+  TEXT = 'text',
+  SELECT = 'select'
```

Comment:
Select er strengt tatt ikke en input type. Virker som URL kan brukes i stedet

Reply:
Brukes for å rendere en selectbox (shapes). Dette siden jeg ikke ønsker å duplisere all kode.